docs/tools.md

```diff
@@ -168,3 +168,54 @@ static get onPaste() {
 > Both `onPaste.handler` and `onPaste.patternHandler` can be `async` or return a `Promise`.
 
 ### Sanitize
+
+CodeX Editor provides [API](https://github.com/codex-team/codex.editor/blob/master/docs/sanitizer.md) to sanitize your fields. We recommend to clean your taint 
+string before you return your data.
```

Comment:
to clean taint strings at the save method or pass sanitizer config to do it automatically.